crates/fsrt/src/main.rs

```diff
@@ -425,10 +344,40 @@ fn main() -> Result<()> {
         .with(EnvFilter::from_env("FORGE_LOG"))
         .init();
     let dirs = std::mem::take(&mut args.dirs);
-    let function = args.function.as_deref();
+
     for dir in dirs {
+        let mut manifest_file = dir.join("manifest.yaml");
+        if !manifest_file.exists() {
+            manifest_file.set_extension("yml");
+        }
+        debug!(?manifest_file);
+
+        let manifest_text = fs::read_to_string(&manifest_file)
+            .into_diagnostic()?
+            .clone();
+
+        let forge_project_from_dir = ForgeProjectFromDir {
+            dir: dir.clone(),
+            manifest_file_content: manifest_text,
+        };
+
         debug!(?dir);
-        scan_directory(dir, function, &args)?;
+        let reporter_result = scan_directory(dir.clone(), &args, forge_project_from_dir);
```

Comment:
```suggestion
        let reporter_result = scan_directory(dir, &args, forge_project_from_dir);
```

Reply:
fixed. thank you .... needed to remove this form the print statemtn below this if i wanted to remove the clone :)